core/src/com/unciv/logic/civilization/diplomacy/DeclareWar.kt

```diff
@@ -43,6 +43,15 @@ object DeclareWar {
         if (otherCiv.isMajorCiv())
             for (unique in civInfo.getTriggeredUniques(UniqueType.TriggerUponDeclaringWar))
                 UniqueTriggerActivation.triggerUnique(unique, civInfo)
+        
+            for (unique in otherCiv.getTriggeredUniques(UniqueType.TriggerUponBeingDeclaredWarUpon))
+                UniqueTriggerActivation.triggerUnique(unique, otherCiv)
+        
+            for (unique in civInfo.getTriggeredUniques(UniqueType.TriggerUponEnteringWar))
+                UniqueTriggerActivation.triggerUnique(unique, civInfo)
+
+            for (unique in otherCiv.getTriggeredUniques(UniqueType.TriggerUponEnteringWar))
+                UniqueTriggerActivation.triggerUnique(unique, otherCiv)
```

Comment:
I could be wrong here, but I believe the indentation is incorrect. Since this doens't apply to `if (otherCiv.isMajorCiv())`, it should likely be back 4 spaces.